### ISSUE TITLE
ADD support for geo: and waze: URI schemes

### ISSUE THREAD

**Issue by allanlaal** [VisualComposer/builder#1919]
currently it seems Visual Composer just blanks hrefs (for IconBoxes atleast) that do not match a preset of (https,http,tel,mailto)

when I add a Link with an url of `geo:37.786971,-122.399677`, the Visual Composer form accepts it, but when its rendered on the final public page, the href value is left blank

https://en.wikipedia.org/wiki/Geo_URI_scheme
https://developers.google.com/waze/deeplinks

**Comment by allanlaal** [comment]
https://github.com/VisualComposer/builder/blob/ff1328872eb6cecd91746ae3a3e80cab193bb0ba/public/sources/attributes/url/UrlDropdownInput.js

https://github.com/VisualComposer/builder/blob/ff1328872eb6cecd91746ae3a3e80cab193bb0ba/public/sources/attributes/url/UrlDropdownInput.js#L80

**Comment by nikitahl** [comment]
Thank you for your request, we will consider it adding to our backlog.

**Comment by allanlaal** [comment]
both Android and iPhone already support these: tapping on a `geo:` or `waze` link will ask the user which Navigation app to use and will remember that choice

**Comment by allanlaal** [comment]
for Googlers:

insert a normal https Link using the Waze maps redirector service:

`https://www.waze.com/ul?zoom=16&lat=59.123455674567&lon=24.12345675676&navigate=yes `

zoom = zoom level. same as in google maps too in the url
lat = latitude. North is + (omitted). South is `-`
lon=longitude. East is + (omitted). West is `-`

going to that URL will check if the phone has Waze installed and will open the link in Waze if it is.
if Waze app is not installed, it will redirect to a waze web map page.

its a tiny bit slower and only works for waze, but one could write/search a similar redirector for Google Maps or whatever combination you like

~ https://developers.google.com/waze/deeplinks

**Comment by nikitahl** [comment]
We have added this feature to our backlog. You can expect it to go live in future updates.
This issue will be closed once PR with implemented code is merged into the master branch.

**Comment by konutis** [comment]
I have added support for waze and geo -> 
https://github.com/VisualComposer/builder/pull/2008

Actually, if you select "custom" in the url dropdown, then you can insert `geo:37.786971,-122.399677` and it will work fine.

But now you will have an option to select geo: and waze: too.

**Comment by nikitahl** [comment]
https://github.com/VisualComposer/builder/pull/2008
